### PR TITLE
Fixes & improvements for the Phoenix Auth reading.

### PR DESCRIPTION
**Pull request by houhoulis**
I'll add explanatory comments in the diff.

### CODE REVIEW

File: reading/phoenix_authentication.livemd [hunk 1]
```diff
@@ -243,7 +249,7 @@ R --> C --> V --> T
 
 <!-- livebook:{"break_markdown":true} -->
 
-Submit the form with any email (real or fake such as `test@test.com`) and any valid password such as `supersecretpassword`. We should never re-use personal passwords, particularly during development where they may not be private.
+Submit the form with any email (real or fake such as `test@example.com`) and any valid password such as `supersecretpassword`. We should never re-use personal passwords, particularly during development where they may not be private.
```
Comment (houhoulis) [inline reading/phoenix_authentication.livemd:252]: "test . com" is a real, live domain that shouldn't be used for any kind of testing (or anything else, by the looks of it). example.com, and some other domains, are reserved by the IANA for testing. https://en.wikipedia.org/wiki/Example.com

File: reading/phoenix_authentication.livemd [hunk 2]
```diff
@@ -1004,10 +1012,8 @@ end
 
 To fix these tests, we need to ensure the created entry belongs to a user, and that the user is logged in.
 
-The `Diary.AccountsFixture` provides a `user_fixture/1` function for creating users, and a `log_in_user/2` function for logging in users.
-
-Import the `Diary.AccountsFixture` to get access to these functions in our tests.
-It's conventional for imports to be grouped together, so put the import with out other `Diary.EntriesFixture`.
+The `Diary.AccountsFixture` module provides a `user_fixture/1` function for creating users. Import the module to get access to this function in our tests.
```
Comment (houhoulis) [inline reading/phoenix_authentication.livemd:1015]: In testing, the `log_in_user/2` function is defined in `test/support/conn_case.ex`, not in the AccountsFixture. At least, that's how it appears to me.

File: reading/phoenix_authentication.livemd (hunk 1 above)
Comment (BrooklinJazz) [reply reading/phoenix_authentication.livemd:252]: Thank you! I did not know!

File: reading/phoenix_authentication.livemd (hunk 2 above)
Comment (BrooklinJazz) [reply reading/phoenix_authentication.livemd:1015]: I believe you are correct!

**Review by BrooklinJazz — APPROVED** [review]
Great improvements!

**Pull request merged**